manifests/init.pp

```diff
@@ -216,7 +216,7 @@
       true    => '--keepdevel',
       default => undef,
     },
-  ])
+  ].filter |$val| { $val =~ NotUndef }
```

Comment:
oh nice, I didn't know that `NotUndef` is a thing!

Reply:
er, maybe I am just mis-reading this, but wouldn't it have been easier to just set the default for $keep_kernel_devel to `''` (empty string)? That is the only potentially undefined value there, right?